.changeset/real-bobcats-train.md

```diff
@@ -0,0 +1,6 @@
+---
+'@rocket.chat/rest-typings': patch
+'@rocket.chat/meteor': patch
+---
+
+Don't show Join default channels option on edit user form and stop accepting this param on API
```

Comment:
We can't just remove the parameter right now in a sudden manner, it's considered a breaking change.